client/src/components/QandA/QuestionsList/Answers/AnswerDetails.jsx

```diff
@@ -9,9 +10,10 @@ const AnswerDetails = ({ name, date, helpfulness, ID }) => {
     <section>
       by {name}, {newDate} &nbsp;
       |&nbsp;
-      Helpful? <span className='clickable'>Yes</span> ({helpfulness}) &nbsp;
+      <HelpfulTracker ID={ID} helpful={helpfulness} usage='answer'/>
+      {/* Helpful? <span className='clickable'>Yes</span> ({helpfulness}) &nbsp;
```

Comment:
saving for later?

Reply:
No. Will remove. Tried something else out but didn't wanna remove previously working code just in case.